tutorial.md

```diff
@@ -105,14 +97,19 @@ import 'package:sky/widgets.dart';
 
 import 'my_tool_bar.dart';
 
-class DemoApp extends App {
+class DemoHome extends StatelessComponent {
   Widget build() {
     return new Center(child: new MyToolBar());
   }
 }
 
 void main() {
-  runApp(new DemoApp());
+  runApp(new App(
```

Comment:
woah, App came out of nowhere here. Needs to be introduced, especially the magic routes argument.